aspnetcore/security/authentication/disable-cookie.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Disable automatic cookie authentication in ASP.NET Core
+author: John0King
+description: Disable automatic cookie authentication in ASP.NET Core
+ms.author: riande
+ms.date: 5/22/2019
+uid: security/authentication/disable-cookie
+---
+# Disable automatic cookie authentication in ASP.NET Core
+
+By [Kirk Larkin](https://github.com/serpent5), [John King](https://github.com/John0King), and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+The [`[Authorize]`](xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute) attribute specifies that access to a controller or action method is restricted to users who meet the authorization requirement. When the user is not authenticated or doesn't have access to the controller or action method:
+
+* An automatic challenge is issued, redirecting the user to the sign-in page.
```

Comment:
This is specific to cookie auth, not [authorize] or controllers. If you were using another auth type like Bearer you'd get 401s.